docs(instrumentation-fs): update readme (#2779)
open-telemetry/opentelemetry-js-contrib: plugins/node/instrumentation-fs/README.md

[![NPM Published Version][npm-img]][npm-url]
[![Apache License][license-image]][license-image]

> This module provides automatic instrumentation for the [`fs`](http://nodejs.org/dist/latest/docs/api/fs.html) module, which should be loaded using the [`@opentelemetry/instrumentation`](https://github.com/open-telemetry/opentelemetry-js/tree/main/packages/opentelemetry-instrumentation) package.

Reviewer: ```suggestion
This module provides automatic instrumentation for the [`fs`](http://nodejs.org/dist/latest/docs/api/fs.html) module, which can be registered using the [`@opentelemetry/instrumentation`](https://github.com/open-telemetry/opentelemetry-js/tree/main/packages/opentelemetry-instrumentation) package.
```

I'm biasing away from "should" here, because it is also completely fine for instrumentation-fs to be loaded/registered by passing it to `NodeSDK` from `@opentelemetry/sdk-node` for example.

Author: I understand, thank you for your support!
